packages/web-components/package.json

```diff
@@ -87,9 +87,9 @@
   },
   "dependencies": {
     "@microsoft/fast-colors": "^5.1.0",
-    "@microsoft/fast-element": "^1.6.0",
-    "@microsoft/fast-foundation": "^2.27.1",
-    "@microsoft/fast-web-utilities": "^5.0.0",
+    "@microsoft/fast-element": "^1.7.0",
```

Comment:
Let's keep this at 1.6.0 if we can.

Reply:
I could not get it to build without errors and run unless I updated fast-element and fast-web-utilities. Maybe it was being caused by the yarn.lock file?

Reply:
Yeah, I'd do a full rebuild with the yarn.lock file. Just check to ensure only one version of FAST Element is included. If you have trouble building, push up the changes and I can look and advise.

Reply:
I think it will pull 2 versions. I was seeing @microsoft/fast-element^1.7.0 and @microsoft/fast-web-utilities^5.0.2 dependencies for @micrsoft/fast-foundation^2.32.0 in the yarn.lock file. Should I upgrade to the last fast-foundation version using fast-element v1.6 and fast-web-utilities v5.0.0?

Reply:
You should be able to manually adjust. Yarn is finicky, but you can adjust the lock file manually if need be to ensure proper resolution.

Reply:
In some cases for deduplication you can run [yarn-deduplicate](https://www.npmjs.com/package/yarn-deduplicate) locally. Just be sure to supply either `--packages` or `--scopes` to avoid deduplicating the entire lock file and breaking things.

Reply:
Got it building that time. I've pushed an update.